interfaces/builtin/desktop_calendar_service.go

```diff
@@ -0,0 +1,138 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const desktopCalendarServiceSummary = `allows communication with Evolution Data Service Calendar`
```

Comment:
The interface names come from @kenvandine's initial branch.  I'm in two minds about this:

1. EDS is put forward as the "system address book/calendar" on GNOME systems (and Unity 7, for older Ubuntu), rather than just being "Evolution's address book".  Granted, it hasn't been universally accepted as such: KDE has their Akonadi service, for instance.

2. this is most definitely a legacy API from a confinement point of view: we're basically looking at a private D-Bus API that was written to replace private CORBA IPC.  If a better API comes along, it isn't clear we'd want to put it on the same snapd interface.

Reply:
@jhenstridge - based on your comments, I feel like this should be eds-contacts and eds-calendar. This makes it very clear that it is for evolution data server's (legacy) APIs and not akonadi. If we felt that this was going to be *the permanent API* for GNOME, then I would suggest desktop-contacts and desktop-calendar with akonadi added to it as people needed it. Once there is a proper public API for contacts and calendar on GNOME, then we can add the desktop-contacts and desktop-calendar interfaces with it. Perhaps one day we'll have portals for contacts and calendar, and that can simply be added to 'desktop' (or the desktop-contacts/desktop-calendar interfaces if that fits better).

If we go with eds-calendar and eds-contacts, then we should not append '-service' IMO since 'eds' stands for 'Evolution Data Server' and thus be redundant. Furthermore, as precedent, unity8-calendar and unity8-contacts went through a lengthy name review and don't use append '-service'.

@niemeyer - can you weigh in on the naming of these interfaces? @jhenstridge, please don't rename until there is agreement on the name.